File: app/src/main/java/com/example/Sleepy/main/modules/onBoarding/domain/OnBoarding.java
package com.example.Sleepy.main.modules.onBoarding.domain;

public class OnBoarding {
    private String Title, Description;
    private int Image;
    private int Animation;

    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        Title = title;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String description) {
        Description = description;
    }

    public int getImage() {
        return Image;
    }

    public void setImage(int image) {
        Image = image;
    }

    public int getAnimation(){ return Animation; }

    public void setAnimationLottie(int animation) {
        Animation = animation;
    }
}

File: app/src/main/java/com/example/Sleepy/main/modules/onBoarding/domain/OnBoarding.kt
package com.example.Sleepy.main.modules.onBoarding.domain

class OnBoarding {
    var title: String? = null
    var description: String? = null
    var image = 0
    var animation = 0
        private set

    fun setAnimationLottie(animation: Int) {
        this.animation = animation
    }
}